ros_ws/src/teleoperation/src/keyboard_controller.cpp

```diff
@@ -0,0 +1,87 @@
+#include "keyboard_controller.h"
+
+KeyboardController::KeyboardController()
+{
+    this->drive_parameters_publisher =
+        this->node_handle.advertise< drive_msgs::drive_param >(
+            TOPIC_DRIVE_PARAMETERS, 1);
+}
+
+void KeyboardController::keyboardLoop()
+{
+    std::cout << "Listening to keyboard..." << std::endl;
+    std::cout << "========================" << std::endl;
+
+    double velocity = 0;
+    double angle    = 0;
+    while (ros::ok())
+    {
+        int key = this->getKeyboardCharacter();
+
+        if (key == KEYCODE_W)
```

Comment:
May replace the `if`s with a single `switch(key){ case A: break; case B: break;}`